.gitignore

```diff
@@ -3,3 +3,5 @@ coverage/
 .nyc_output
 package-lock.json
 *.swp
```

Comment:
Just a simple question... why exclude the contents of dist? I think this needs to be included if this package will be published. Otherwise, the users have to always run "npm run build"

Reply:
Opps, my mistake, just added the dist folder there and removed /dist from .gitignore